docs/docs/12-context.md

```diff
@@ -64,7 +64,7 @@ var Button = React.createClass({
   },
   render: function() {
     return (
-      <button style={{'{{'}}background: this.context.color}}>
+      <button style={{background: this.context.color}}>
```

Comment:
This (and the others like it you changed) needs to be written as it was, as Jim said. It may render correctly in the GH preview but will not render correctly when using Jekyll because `{{` is an opening tag for Liquid templating which is used for the actual website..